src/controllers/productController.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+import { deleteProductById, getProductById } from '../services/productServices';
+  
+  /**
+   * Product Controller class
+   */
+  export default class productController {
+  
+    /**
+     * delete producr
+     * @param {Request} req - Express request object
+     * @param {Response} res - Express response object
+     */
+
+    static async deleteProduct(req: Request, res: Response) {
+      try {
+        const productId = req.params.itemId;
+        const { id } = req.user;
+        const product = await getProductById(id, productId)
+        if(product){
+          const result: number = await deleteProductById(id, productId)
+          if (result !== 0) {
+            return res.status(200).json({message: "item deleted", productDeleted: product})
+          }

```

Comment:
How the deleted Product can be displayed??
May be you can return only message instead